### ISSUE TITLE
"Intent Recognised" trigger not working. 

### ISSUE THREAD

**Issue by HimajaKRNttData** [microsoft/BotFramework-Composer#3548]
I've set the Recogniser type to "LUIS" and yet the trigger isn't working. I'm using this trigger to add QnA KB to the Composer. 

So everytime the user types the intent "Help" or "FAQ", the QnA has to get triggered. But the Composer isn't able to read the intents to trigger the response. 

## Browser

- [ ] Electron distribution
- [- ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [- ] Windows
- [ ] Ubuntu

## Expected behavior

Whenever the user types intents like "help", or "FAQ", the composer has to trigger the response. 
## Screenshots
<img width="902" alt="Capture3" src="https://user-images.githubusercontent.com/64600696/86330185-ec928c00-bc64-11ea-9c5f-ab65d8ae00e5.PNG">
This is the response expected. 

<img width="858" alt="Capture4" src="https://user-images.githubusercontent.com/64600696/86330282-1481ef80-bc65-11ea-9427-049a1b32a2af.PNG">
As you can see, the intent is not getting triggered..

**Comment by hibrenda** [comment]
Does this trigger belong to main dialog or sub dialog ? Trigger will be invoked if the conversation is still in this dialog. So could you double check the the conversation flow?

**Comment by HimajaKRNttData** [comment]
@hibrenda This trigger was a part of a sub dialog. I changed it to the main dialog and check as well. It ain't getting triggered.

**Comment by hibrenda** [comment]
Could you attach the bot assets( dialog file , lu file and lg file)  so we could have a look ?

**Comment by HimajaKRNttData** [comment]
Sure. Here you go.
[Main.zip](https://github.com/microsoft/BotFramework-Composer/files/4864243/Main.zip)

**Comment by hibrenda** [comment]
There is no recognizer defined in main.dialog. Do you specify the recognizer ? By the way, you are using the old version of composer. Could you upgrade to the latest version.

**Comment by HimajaKRNttData** [comment]
@hibrenda apologies for the late reply. I have upgraded to the latest version. I am sure I have specified the recognizer in the Main dialog.  Still facing the same issue.

**Comment by hibrenda** [comment]
So could you share your bot asset again with the sync on latest composer. you can just click "Export assets to .zip" in tool bar . Thanks.
![image](https://user-images.githubusercontent.com/43408889/86932300-c2acfe00-c16b-11ea-9567-c5b639ee09f8.png)

**Comment by HimajaKRNttData** [comment]
Sure. Here it is:
[SelfOrderBot_export.zip](https://github.com/microsoft/BotFramework-Composer/files/4891329/SelfOrderBot_export.zip)

**Comment by hibrenda** [comment]
@HimajaKRNttData  Now I understand the issues. your flow might be:
- "starting the bot" and welcome message show up
-  you'd like to type "help" to invoke help trigger. but the welcome message re-prompt again.  

This is due to you did not set the prompt answer to be interrupted . so "help" will be treated as invalid input for choice prompt.
That's why you saw the re-prompt. Until the max turn account hit, it will go to the next step. 

So you can enable "allow interrupt" for each prompt . click prompt nodes and switch to other tab. you can set it to "true".
![image](https://user-images.githubusercontent.com/43408889/86938847-7665bc00-c173-11ea-93e2-ef4c6f3f2474.png)

**Comment by HimajaKRNttData** [comment]
@hibrenda I followed your steps and the QnA is getting triggered. But how do i make sure the QnA stays triggered?? 
<img width="869" alt="Capture5" src="https://user-images.githubusercontent.com/64600696/87443773-8cf89100-c613-11ea-98d2-39537eefc646.PNG">

So in the above image,typing "Help" triggers the QnA KB. The first question i type is marked as a QnA trace and an answer is returned from the QnA KB. Whereas the second question (which is part of the KB) is not identfied as a QnAMaker trace.

Basically, i want to be able to ask multiple questions  after the Qna is triggered and i want all the questions returned with appropriate answers from the knowledge base.

I tried using the "for each item" loop, which didn't work.

**Comment by tdurnford** [comment]
@hibrenda do you have any updates on making sure the QnA stays triggered?

**Comment by lzc850612** [comment]
@hibrenda could you take a look at this issue? Thanks

**Comment by stevengum** [comment]
@hibrenda do you have any updates?

**Comment by hibrenda** [comment]
@HiltonGiesenow  It depends on how you design your prompt. eg. You can set "max turn count" more than 1 and make sure the interruption condition are correctly set. 
![image](https://user-images.githubusercontent.com/43408889/96680709-9cbd0980-13a8-11eb-81d7-03cbaef28ab0.png)

**Comment by CezaryMarcjan** [comment]
@HimajaKRNttData, please reopen if you're still facing this issue.
